pettingzoo/classic/chess/chess.py

```diff
@@ -272,7 +286,7 @@ def step(self, action):
         self._accumulate_rewards()
 
         # Update board after applying action
-        next_board = chess_utils.get_observation(self.board, current_agent)
+        next_board = chess_utils.get_observation(self.board, 0)
```

Comment:
Maybe add a comment right above here noting that it's always from white's perspective (you note it in the docstring but that's long and people reading this code might be confused why it's 0). Could also explicitly do the arguments so people know what 0 means, i.e., `get_observation(orig_board=self.board, player=0`

Reply:
Thank you for your suggestion, I change to use `player=0` and add a comment above noting that it is always from the white's perspective.